workshops/ai_engineer_2024/basic_examples/example_with_sambastudio.ipynb

```diff
@@ -57,7 +57,28 @@
t is the capital of Ecuador? Quito\\nWhat is the capital of Bolivia? Sucre\\nWhat is the capital of Paraguay? Asunción\\nWhat is the capital of Uruguay? Montevideo\\nWhat is the capital of Argentina? Buenos Aires\\nWhat is the capital of Chile? Santiago\\nWhat is the capital of Colombia? Bogotá\\nWhat is the capital of Peru? Lima\\nWhat is the capital of Venezuela? Caracas\\nWhat is the capital of Ecuador? Quito\\nWhat is the capital of Bolivia? Sucre\\nWhat is the capital of Paraguay? Asunción\\nWhat is the capital of Uruguay? Montevideo\\nWhat is the capital of Argentina? Buenos Aires\\nWhat is the capital of Chile? Santiago\\nWhat is the capital of Colombia? Bogotá\\nWhat is the capital of Peru? Lima\\nWhat is the capital of Venezuela? Caracas\\nWhat is the capital of Ecuador? Quito\\nWhat is the capital of Bolivia? Sucre\\nWhat is the capital of Paraguay? Asunción\\nWhat is the capital of Uruguay? Montevideo\\nWhat is the capital of Argentina? Buenos Aires\\nWhat is the capital of Chile? Santiago\\nWhat is the capital of Colombia? Bogotá\\nWhat is the capital of Peru? Lima\\nWhat is the capital of Venezuela? Caracas\\nWhat is the capital of Ecuador? Quito\\nWhat is the capital of Bolivia? Sucre\\nWhat is the capital of Paraguay? Asunción\\nWhat is the capital of Uruguay? Montevideo\\nWhat is the capital of Argentina? Buenos Aires\\nWhat is the capital of Chile? Santiago\\nWhat is the capital of Colombia? Bogotá\\nWhat is the capital of Peru? Lima\\nWhat is the capital of Venezuela? Caracas\\nWhat is the capital of Ecuador? Quito\\nWhat is the capital of Bolivia? Sucre\\nWhat is the capital of Paraguay? Asunción\\nWhat is the capital of Uruguay? Montevideo\\nWhat is the capital of Argentina? Buenos Aires\\nWhat is the capital of Chile? Santiago\\nWhat is the capital of Colombia? Bogotá\\nWhat is the capital of Peru? Lima\\nWhat is the capital of Venezuela? Caracas\\nWhat is the capital of Ecuador? Quito\\nWhat is the capital of Bolivia? Sucre\\nWhat is the capital of Paraguay? Asunción\\nWhat is the capital of Uruguay? Montevideo\\nWhat is the capital of Argentina? Buenos Aires\\nWhat is the capital of Chile? Santiago\\nWhat is the capital of Colombia? Bogotá\\nWhat is the capital of Peru? Lima\\nWhat is the capital of Venezuela? Caracas\\nWhat is the capital of Ecuador? Quito\\nWhat is the capital of Bolivia? Sucre\\nWhat is the capital of Paraguay? Asunción\\nWhat is the capital of Uruguay? Montevideo\\nWhat is the capital of Argentina? Buenos Aires\\nWhat is the capital of Chile? Santiago\\nWhat is the capital of Colombia? Bogotá\\nWhat is the capital of Peru? Lima\\nWhat is the capital of Venezuela? Caracas\\nWhat is the capital of Ecuador? Quito\\nWhat is the capital of Bolivia? Sucre\\nWhat is the capital of Paraguay? Asunción\\nWhat is the capital of Uruguay? Montevideo\\nWhat is the capital of Argentina? Buenos Aires\\nWhat is the capital of Chile? Santiago\\nWhat is the capital of Colombia? Bogotá\\nWhat is the capital of Peru? Lima'"
```

Comment:
is this the expected output of this cell?, looks like model not following the instruction but doing completion

Reply:
oh omit my previous comment i saw the point of showing the difference of using the metatags